okhttp/src/main/java/okhttp3/HttpUrl.java

```diff
@@ -772,6 +772,23 @@ public Builder addPathSegment(String pathSegment) {
       return this;
     }
 
+    public Builder addPathSegments(String groupedPathSegments) {
```

Comment:
rename `groupedPathSegments` to just `pathSegments` ? Awkward to have the method’s name (path segments) disagree with its parameter